.github/workflows/test.yml

```diff
@@ -1,8 +1,16 @@
 name: "Tests"
+
 on: pull_request
 
+env:
+  OTP_VERSION: 23.0.2
+  ELIXIR_VERSION: 1.10.4
+  PHOENIX_VERSION: 1.5.5
+  NODE_VERSION: 12
+  PROJECT_PATH: "sample_project"
```

Comment:
This does not look like a path e.g. `/user/test/sample_project`. How about rename this to `PROJECT_DIRECTORY` or `PROJECT_ROOT`?

Reply:
How about `PROJECT_PATH: "/sample_project"` @olivierobert 🤔

Reply:
Fixed in https://github.com/nimblehq/elixir-templates/pull/24/commits/84a7e51b0a5abc1ddde3a9b0aee64f1c1d6700fb